lib/sidekiq/metrics/stats.rb

```diff
@@ -0,0 +1,56 @@
+require 'sidekiq/api'
+
+module Sidekiq
+  module Metrics
+    class Stats
+      def initialize(
+        influxdb_client:,
+        series_name: 'sidekiq_stats',
+        retention_policy: nil,
+        tags: {},
+        clock: -> { Time.now.to_f }
+      )
+        @influxdb = influxdb_client
+        @series = series_name
+        @retention = retention_policy
+        @tags = tags
+        @clock = clock
+      end
+
+      def call
+        stats ||= ::Sidekiq::Stats.new.instance_variable_get(:@stats)
```

Comment:
Is this really necessary? There should be a reason for this variable's inaccessibility.

Reply:
It's again copypasta